tsfresh/feature_selection/feature_selector.py

```diff
@@ -116,8 +117,12 @@ def check_fs_sig_bh(X, y, settings=None):
 
     # Helper function which wraps the _calculate_p_value with many arguments already set
     f = partial(_calculate_p_value, y=y, settings=settings, target_is_binary=target_is_binary)
-    results = pool.map(f, [X[feature] for feature in df_features['Feature']], chunksize=settings.chunksize)
-    p_values_of_features = pd.DataFrame(results)
+
+    total_number_of_features = len(X.columns)
+    results = tqdm(pool.imap_unordered(f, [X[feature] for feature in df_features['Feature']]),
```

Comment:
Can we use the same function ```_calculate_best_chunksize``` from ```extraction.py``` to calculate the chunksize? If yes, it may be worth to move it to utililties.